spec/template.dd

```diff
@@ -53,7 +53,19 @@ $(GNAME TemplateParameter):
         $(I TemplateParameter) in case one is not supplied.
     )
 
-$(H2 $(LNAME2 explicit_tmp_instantiation, Explicit Template Instantiation))
+
+$(H2 $(LNAME2 template_instantiation, Template Instantiation))
+
+    $(P A template must be instantiated before use. This means passing one or
```

Comment:
templates instantiations can have no arguments

Reply:
Sorry, I'm not sure what you mean. Instantiating does take arguments.

Reply:
`temp!()()`

Reply:
I didn't write passing arguments to an instantiation though, I said *passing arguments to the template*. I don't understand what's wrong.

Reply:
Oh I see! I changed it to `This usually means passing one or more`...

Reply:
A specification should be precise, I would avoid words like 'usually' or 'often'. How about "This means passing an argument list".

Reply:
@dkorpel I've now added that phrasing.